services/QuillLMS/client/app/bundles/Evidence/components/studentView/container.tsx

```diff
@@ -94,6 +94,14 @@ export const StudentViewContainer = ({ dispatch, session, isTurk, location, acti
     4: 0
   })
 
+  React.useEffect(() => {
+    dispatch(setActiveStepForSession(activeStep));
+  }, [activeStep])
```

Comment:
We should have a single source of truth here - so rather than updating the local `activeStep` and then calling `setActiveStepForSession` when it changes, we should just call `setActiveStepForSession` every time we'd update it locally and use the one passed down from the reducer as our `activeStep` in this file.